packages/table/src/table.tsx

```diff
@@ -450,6 +450,18 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
+    public resizeRowsBasedOnColumnHeight(columnIndex: number) {
```

Comment:
I just mean that for our API where have a method for just one of rows or columns, users usually ask for the other one.